src/ui/devtools/style.less

```diff
@@ -65,6 +65,7 @@ header {
     outline: none;
     padding: @size-control-inner / 2 - @size-text-normal / 2;
     resize: none;
+    tab-size: 4;
```

Comment:
We are not using Tab symbol in config, so there should be no need for this property.